src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -2120,6 +2151,15 @@ internal override BaseClientSessionTransportManager CreateClientSessionTransport
                 startInfo.ArgumentList.Add(string.Format(CultureInfo.InvariantCulture, @"-p {0}", this.Port));
             }
 
+            // pass "-o option=value" command line argument to ssh if options are provided
+            if (this.Options != null) 
+            {
+                foreach (DictionaryEntry pair in this.Options) 
+                {
+                    startInfo.ArgumentList.Add(string.Format(CultureInfo.InvariantCulture, @"-o {0}={1}", pair.Key, pair.Value));
```

Comment:
Since we get this through public API should we make additional validations?
I mean that `Key` and `Value` could contain composite values.
/cc @PaulHigin

Reply:
I see your point, right now the Format method will just try to convert whatever is passed in to a string, would we want to handle composite values any other way? 

I also could try to add some validation to ensure that we're only passing options that OpenSSH would consider valid, but that would be a lot to maintain as well. Please let me know your thoughts.